Update Notes-CaaSP.adoc

rknightion opened the pull request:
Added in unsupported and for dev use only note

Commit:
Update Notes-CaaSP.adoc

Added in unsupported and for dev use only note

mook-as reviewed (approved):
It might be good to point people to the _correct_ way to do it as well ( https://www.suse.com/documentation/cloud-application-platform-1/book_cap_deployment/data/book_cap_deployment.html ); and for deploying CaaSP, point them at… heat templates? Somewhere?

Commit:
Update Notes-CaaSP.adoc

Added additional links to official documentation and tidied up some product names

Commit:
Update Notes-CaaSP.adoc

Minor fixes

cjschroder commented:
Sorry @rssfed23 but I don't understand this, Section 3 says-- "Please note that the procedure detailed below is NOT a supported way to deploy SUSE Containers as a Service Platform" and then the section ends. What procedure is this referring to? Look at https://www.suse.com/releasenotes/x86_64/SUSE-CAP/1/#sec.caasp; Maybe this should be a warning at the beginning of the relevant section.

Also, Release Notes really shouldn't contain entire chapters. They're for changes, known issues, and workarounds. Anyone inspired to write chapters please contribute to the CAP Deployment Guide :)  You don't even have to apply DocBook XML markup, just give me the text.

cjschroder commented:
I removed the Terraform/OpenStack section https://github.com/SUSE/cap-release-notes/pull/38

Links to the official docs for customers are :+1:  Thanks @rssfed23 
See https://github.com/SUSE/cap-release-notes/pull/38 for how we are organizing internal docs and customer docs
